sklearn/decomposition/benchmark_nmf.py

```diff
@@ -0,0 +1,116 @@
+
```

Comment:
This file should be moved to the "benchmarks" folder, in the root of the scikit-learn project.